scripts/db_migrator.py

```diff
@@ -1249,6 +1261,7 @@ def version_202505_01(self):
         master branch until 202505 branch is created.
         """
         log.log_info('Handling version_202505_01')
+        self.migrate_flex_counter_delay_status_removal()
```

Comment:
I think you are missing set version call to `version_202505_02` and a placeholder function for `version_202505_02`.

Reply:
Before your change the version is already set to `version_202505_01`, so with your change (a new migration), the version should be moved to `version_202505_02`, and that's where `migrate_flex_counter_delay_status_removal` will get called. Please check previous pr examples.

Reply:
@vaibhavhd I am confused by the comment on this function:

```
Version 202505_01, this version should be the final version for
        master branch until 202505 branch is created.
```

So, I didn't create a new version.

Reply:
@vaibhavhd unless someone is doing master -> master upgrades no need to create intermediate versions.
This approach makes too many conflicts. Before branch is out it is easier to put all in one version handler and then split versions.

@yxieca Could you please advise how to handle versions in db_migrator when the branch is not out yet?